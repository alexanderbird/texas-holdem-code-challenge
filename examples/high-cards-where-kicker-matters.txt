2C
Winnie TC 9C 8H 7H
Foster TD 9H 8D 6H
Alvin TS 9D 7C 4C
Lowrance TH 8C 4H 3D
Creigh KC QC JH TD
Cassey KH JC TH 9D
Kalinda KD TC 9S 8S
