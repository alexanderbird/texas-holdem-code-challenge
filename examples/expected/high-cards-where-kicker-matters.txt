1 Creigh Highcard King (kicker Queen)
2 Cassey Highcard King (kicker Jack)
3 Kalinda Highcard King (kicker 10)
4 Winnie Highcard 10 (kicker 9 8 7)
5 Foster Highcard 10 (kicker 9 8 6)
6 Alvin Highcard 10 (kicker 9 7)
7 Lowrance Highcard 10 (kicker 8)
